src/Chapter1.hs

```diff
@@ -520,7 +521,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if (abs x) - (abs y) < 0 then x else y
```

Comment:
`()` are redundant here. I see how it can be confusing to see when to use them and when not to use, especially in the beginning. But it's usually better to remove redundant brackets for cleaner code :broom: 

Also, I think comparison of `abs`olute values would work better here instead of subtraction. There are could be edge cases where this doesn't work. And generally, looks a little bit shorter 🙂 
```suggestion
closestToZero x y = if abs x < abs y then x else y
```

Reply:
Cool, thanks! For sure your suggested solution is better, but out of curiosity trying to find _edge cases where this doesn't work_ for my solution and could not came up with even one 🤔 Could you provide some examples?

Reply:
I was thinking about the lowest and greatest `Int` values but my quick local testing wasn't able to find a case where this function breaks, so it seems actually working 👍🏻